src/OSPSuite.Core/Domain/Populations/IndividualValuesCache.cs

```diff
@@ -32,7 +44,7 @@ public void Add(IndividualValues individualValues)
       /// <summary>
       ///    Returns the number of individuals
       /// </summary>
-      public virtual int Count => ParameterValuesCache.Count;
```

Comment:
This was quite confusing. The count here was the not the number of items in the cache but the number of values in the first item of the cache (e.g. number of individuals). Like that, this is much clearner